<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindowClass</class>
  <widget class="QMainWindow" name="MainWindowClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>500</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>考古绘图辅助</string>
    </property>
    <widget class="QWidget" name="centralWidget"/>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>800</width>
          <height>23</height>
        </rect>
      </property>
      <widget class="QMenu" name="menuFile">
        <property name="title">
          <string>文件</string>
        </property>
        <addaction name="actionImport"/>
        <addaction name="actionSave"/>
        <addaction name="actionSaveTexture"/>
      </widget>
      <widget class="QMenu" name="menuOptions">
        <property name="title">
          <string>选项</string>
        </property>
        <addaction name="actionCoordidateSystem"/>
        <addaction name="actionSymmetryAxis"/>
        <addaction name="actionStartUnfold"/>
      </widget>
      <addaction name="menuFile"/>
      <addaction name="menuOptions"/>
    </widget>
    <widget class="QToolBar" name="mainToolBar">
      <property name="baseSize">
        <size>
          <width>11</width>
          <height>11</height>
        </size>
      </property>
      <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <attribute name="toolBarArea">
        <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
        <bool>false</bool>
      </attribute>
      <action name="actionImport">
        <property name="icon">
          <iconset>
            <normaloff>Icons/open.png</normaloff>Icons/open.png
          </iconset>
        </property>
        <property name="text">
          <string>打开</string>
        </property>
      </action>
      <action name="actionSave">
        <property name="icon">
          <iconset>
            <normaloff>Icons/save.png</normaloff>Icons/save.png
          </iconset>
        </property>
        <property name="text">
          <string>保存</string>
        </property>
      </action>
      <action name="actionSaveTexture">
        <property name="icon">
          <iconset>
            <normaloff>Icons/save.png</normaloff>Icons/save.png
          </iconset>
        </property>
        <property name="text">
          <string>保存纹理</string>
        </property>
      </action>
      <action name="actionCoordidateSystem">
        <property name="icon">
          <iconset>
            <normaloff>Icons/reload.png</normaloff>Icons/reload.png
          </iconset>
        </property>
        <property name="text">
          <string>坐标系</string>
        </property>
      </action>
      <action name="actionSymmetryAxis">
        <property name="icon">
          <iconset>
            <normaloff>Icons/reload.png</normaloff>Icons/reload.png
          </iconset>
        </property>
        <property name="text">
          <string>对称轴</string>
        </property>
      </action>
      <action name="actionStartUnfold">
        <property name="icon">
          <iconset>
            <normaloff>Icons/start.png</normaloff>Icons/start.png
          </iconset>
        </property>
        <property name="text">
          <string>开始</string>
        </property>
      </action>
      <addaction name="actionImport"/>
      <addaction name="actionSave"/>
      <addaction name="actionSaveTexture"/>
      <addaction name="separator"/>
      <addaction name="actionCoordidateSystem"/>
      <addaction name="actionSymmetryAxis"/>
      <addaction name="actionStartUnfold"/>
    </widget>
    <widget class="QStatusBar" name="statusBar"/>
    <action name="actionImport">
      <property name="icon">
        <iconset>
          <normaloff>Icons/open.png</normaloff>Icons/open.png
        </iconset>
      </property>
      <property name="text">
        <string>打开</string>
      </property>
    </action>
    <action name="actionSave">
      <property name="icon">
        <iconset>
          <normaloff>Icons/save.png</normaloff>Icons/save.png
        </iconset>
      </property>
      <property name="text">
        <string>保存</string>
      </property>
    </action>
    <action name="actionSaveTexture">
      <property name="icon">
        <iconset>
          <normaloff>Icons/save.png</normaloff>Icons/save.png
        </iconset>
      </property>
      <property name="text">
        <string>保存纹理</string>
      </property>
    </action>
    <action name="actionCoordidateSystem">
      <property name="checkable">
        <bool>true</bool>
      </property>
      <property name="checked">
        <bool>false</bool>
      </property>
      <property name="icon">
        <iconset>
          <normaloff>Icons/reload.png</normaloff>Icons/reload.png
        </iconset>
      </property>
      <property name="text">
        <string>坐标系</string>
      </property>
    </action>
    <action name="actionSymmetryAxis">
      <property name="checkable">
        <bool>true</bool>
      </property>
      <property name="checked">
        <bool>false</bool>
      </property>
      <property name="icon">
        <iconset>
          <normaloff>Icons/reload.png</normaloff>Icons/reload.png
        </iconset>
      </property>
      <property name="text">
        <string>对称轴</string>
      </property>
    </action>
    <action name="actionStartUnfold">
      <property name="icon">
        <iconset>
          <normaloff>Icons/start.png</normaloff>Icons/start.png
        </iconset>
      </property>
      <property name="text">
        <string>开始</string>
      </property>
    </action>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <resources>
    <include location="mainwindow.qrc"/>
  </resources>
  <connections/>
</ui>
